grudge/discretization.py

```diff
@@ -80,7 +80,10 @@ class DiscretizationCollection:
 
     def __init__(self, array_context: ArrayContext, mesh: Mesh,
                  order=None,
-                 discr_tag_to_group_factory=None, mpi_communicator=None,
+                 discr_tag_to_group_factory=None,
+                 volume_discr=None,
```

Comment:
I remember this came up recently: For surface discretizations, "volume" is  a bit of a misnomer.

Reply:
related: https://github.com/inducer/grudge/issues/103 -- we'll probably need to introduce a `BASE_DISCR` tag or something like that